### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by jansende**

### CODE REVIEW

**Comment by jansende** [comment]
Hope this is not too much to look through and merge 😇

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 1]
```diff
@@ -171,12 +171,28 @@ std::cout << "Average speed: "
...
+    return avg_speed(t, d) / std::units::constants::speed_of_light; 
+}
+```
+Should fail to compile with:
+```text
+example.cpp: in function 'constexpr std::units::Velocity auto avg_speed(std::units::Length auto d, std::units::Time auto t)'
+example.cpp: error: t is not a Length.
+example.cpp: error: d is not a Time.
+```
+
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Hmm, I am not sure about the `lightspeed_limit()` example. It is not clear what a `std::units::percent` is. Also, it mentions `std::units::constants::speed_of_light` constant. Let's not scare people as of yet and not start the discussion if `std::units::constants` is the right place to have such values. There will be another paper with the description of the library after this one. Also, it mentions `example.cpp: error: t is not a Length.`. But why the error happen (no invalid input presented in the paper). Another point is that none compiler on the market is able to produce such an error as of now.

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 2]
```diff
@@ -205,21 +221,27 @@ ISO 80000-1:2009(E) Quantities and units - Part 1: General
...
+- The division of the concept ‘quantity’ into several kinds is to some extent arbitrary. For 
+    example, the quantities diameter, circumference, and wavelength are generally considered
+    to be [=quantities=] of the same kind, namely, length.
 - [=Quantities=] of the same kind within a given [=system of quantities=] have the same quantity
     [=dimension=]. However, [=quantities=] of the same [=dimension=] are not necessarily of
-    the same kind.
+    the same kind. For example, the absorbed dose and the dose equivalent have the same
+    dimension. However, the former measures the absolute amount of radiation one receives
+    wherase the latter is a weighted measurement taking into account the kind of radiation on 
+    was exposed to.
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Those definitions are taken straight from ISO 80000 and I wouldn't change or amend them in any form.

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 3]
```diff
@@ -205,21 +221,27 @@ ISO 80000-1:2009(E) Quantities and units - Part 1: General
     which are quantities, is also considered to be a quantity.
 - The concept ’quantity’ may be generically divided into, e.g. ‘physical quantity’,
     ‘chemical quantity’, and ‘biological quantity’, or ‘base quantity’ and ‘derived quantity’.
+- Examples of quantities are: mass, length, density, magnetic field strength, etc.
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Same as below

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 4]
```diff
@@ -205,21 +221,27 @@ ISO 80000-1:2009(E) Quantities and units - Part 1: General
...
+    dimension. However, the former measures the absolute amount of radiation one receives
+    wherase the latter is a weighted measurement taking into account the kind of radiation on 
+    was exposed to.
 
 <dfn lt="system of quantities|system">system of quantities, system</dfn>
 
 - Set of [=quantities=] together with a set of non-contradictory equations relating those
     [=quantities=].
+- Examples of systems of quantities are: the International System of Quantities, the Imperial
+    System, etc.
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Same as above. In general, we can copy-paste more from the ISO paper but we shouldn't add our own prose here.

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 5]
```diff
@@ -295,13 +317,16 @@ unit</dfn>
...
 - In each coherent [=system of units=], there is only one base unit for each [=base quantity=].
 - A base unit may also serve for a [=derived quantity=] of the same [=quantity dimension=].
+- For example, the ISQ has the base units of: metre, kilogram, second, Ampere, Kelvin, mole,
+    and candela.
 
 <dfn>derived unit</dfn>
 
-- Measurement unit for a [=derived quantity=]
+- Measurement unit for a [=derived quantity=].
+- For example, in the ISQ Newton, Pascal, and katal are derived units.
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Same

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 6]
```diff
@@ -1570,22 +1647,31 @@ Polls {#polls}
...
+     sometimes doing unnecessary conversions)?
+10. Do we agree with Kelvins only support for temperature and verbose conversion functions
+      for other [=units=] and absolute temperatures?
+11. Which option of UDLs do we prefer ([[#integral-udl]])?
+12. Should we provide `seconds<int>` or stay with `quantity<second, int>`?
+13. Should we provide support for dimensionless quantities?
+14. Should American spelling be provided? (`meter` vs. `metre`, `ton` vs. `tonne`, ...)
+15. Do we want to introduce a dedicated system type?
+16. Should affine types be provided? (relative vs absolute)
+17. Should ISO 80000-1:2009 units be provided? If yes, how should updates to the ISO standard
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: What do you mean by this? Do you mean what happens when for example already implemented conversion factor is updated in ISO 80000?

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 7]
```diff
@@ -1570,22 +1647,31 @@ Polls {#polls}
...
+      be handled? (Separate namespaces?)
+18. Should constants be provided? If yes, how should updates to the constants be handled?
+      (Separate namespaces?)
+19. Should the quantity representation be limited to built-in types?
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: The answer is no. There is a strong desire of the industry to have custom Rep here. So let's remove this question.

File: src/1215_a_cpp_approach_to_physical_units.bs [hunk 8]
```diff
@@ -1594,7 +1680,7 @@ Acknowledgments {#acknowledgements}
 Special thanks and recognition goes to [Epam Systems](http://www.epam.com) for supporting my
 membership in the ISO C++ Committee and the production of this proposal.
 
-I would also like to thank Jan Sende for his contributions to the `mp-units` library and
+I would also like to thank Jan A. Sende for his contributions to the `mp-units` library and
```
Comment (mpusz) [inline src/1215_a_cpp_approach_to_physical_units.bs]: Sorry, I didn't know :-)

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 2 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Okay. I didn't check and assumed that you made slight adjustments. My bad!

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 8 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: No problem, I didn't tell.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 2 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Nevertheless, I would add examples at the bottom. For people new with the topic, it is already hard enough to understand.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 6 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Yes. Usually these changes are backwards compatible, but they don't have to be. Sometimes some of the last digits in constants change and thus all conversions relying on these constants change as well. While not important for most people, that's a problem one should think of.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 7 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Agreed.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 6 above)
Comment (mpusz) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Do we depend only on the ISO document or maybe we should follow some other SI-specific papers produced by Bureau International des Poids et Mesures (BIPM). This starts to be a complex issue :-(

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 1 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Hmm... Maybe you are right about skipping this one.

The `example.cpp: error: t is not a Length.` message is right.  I switched the arguments with the signature of the example above: `avg_speed(Length, Time)`

If you would still like an example, I can think about another one.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 6 above)
Comment (jansende) [reply src/1215_a_cpp_approach_to_physical_units.bs]: Unfortunately yes. However, I don't think this needs to be solved soon. The actual conversion factors and constants are a important implementation detail, but still a implementation detail. Settling on one of the standards later should be fine.

This means too, that we probably shouldn't commit to any of the standards right now. Doing so without enough discussion on the topic, will lead to a lot of changes later on.

File: src/1215_a_cpp_approach_to_physical_units.bs (hunk 1 above)
Comment (mpusz) [reply src/1215_a_cpp_approach_to_physical_units.bs]: I think that this example is not needed. ISO committee members are already familiar with Concepts so for them, this case just duplicates the first example as we do not have to explicitly say to them again that wrong types will fail with a nices error logs ;-) This subject is covered later on anyway too.

**Pull request merged**